plugins/woocommerce/package.json

```diff
@@ -222,6 +222,7 @@
 				"node_modules/@woocommerce/e2e-core-tests/CHANGELOG.md",
 				"node_modules/@woocommerce/api/dist/",
 				"node_modules/@woocommerce/admin-e2e-tests/build",
+				"node_modules/@woocommerce/classic-assets/build",
```

Comment:
Out of curiosity? What's the purpose of that entry? Was it added automatically? If not, what does it solve?

Reply:
This was added automatically when I merged `trunk` into this branch. I am not sure if I should keep this automatically generated file 🤔

Reply:
I went ahead and removed it in https://github.com/woocommerce/woocommerce/pull/43314/commits/18b7ebf19d7f5e8e3de15879e59bd76da686fa0f as it doesn't make sense to have this change in this PR.